usr/src/lib/libuutil/common/libuutil.h

```diff
@@ -245,7 +245,7 @@ void uu_list_pool_destroy(uu_list_pool_t *);
  * usage:
  *
  *	foo_t *a;
- *	a = malloc(sizeof(*a));
+ *	a = malloc(sizeof (*a));
```

Comment:
This was added when I did a find / replace to check for places where I had not put a space there. Although it is not related to encryption, this is probably a good change to have merged anyway for adherence to the style guide.